ydb/docs/en/core/integrations/toc_i.yaml

```diff
@@ -1,3 +1,5 @@
 items:
 - name: Log records collection in a Kubernetes cluster using FluentBit and YDB
   href: fluent-bit.md
+- name: goose
```

Comment:
It is suggestion from @blinkov. In the first version of article had a long name.
I agree with Ivan, because list, search and choose some article by long title in `Integrations` folder is too difficulty, but easy with short keywords

Reply:
The article of fluentbit will renamed in TOC from long to short